app/templates/components/conversations/conversation-thread.hbs

```diff
@@ -1,37 +1,31 @@
 <div class="conversation-thread-inner">
   <div class="conversation-thread__messages">
     {{#if conversation.message.isLoaded}}
-      {{conversations/conversation-part-comment
-        author=conversation.message.author
-        body=conversation.message.body
-        readAt=conversation.message.readAt
-        sentAt=conversation.message.insertedAt
-      }}
-    {{/if}}
-    {{#each sortedConversationParts as |conversationPart|}}
-      {{#if conversationPart.isLoaded}}
-        {{#if (eq conversationPart.partType 'closed')}}
-          {{conversation/conversation-part-closed
-            author=conversationPart.author
-            closedAt=conversationPart.insertedAt
-          }}
-        {{else if (eq conversationPart.partType 'reopened')}}
-          {{conversations/conversation-part-comment
-            author=conversationPart.author
-            body=conversationPart.body
-            readAt=conversationPart.readAt
-            sentAt=conversationPart.insertedAt
-          }}
-        {{else}}
-          {{conversations/conversation-part-comment
-            author=conversationPart.author
-            body=conversationPart.body
-            readAt=conversationPart.readAt
-            sentAt=conversationPart.insertedAt
-          }}
+      {{#each sortedConversationParts as |conversationPart|}}
+        {{#if conversationPart.isLoaded}}
+          {{#if (can "administer project" project)}}
```

Comment:
Your template does not have access to a `project` attribute. It has access to a `conversation`, which is related to a `project`

```Handlebars
{{#if (can "administer project" conversation.project)}}
```